src/constants/notification-templates.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
Just a suggestion. can we maintain these objects like:
```json
{
  "buyer/seller-signup":{
    "sms": {
      "message": "Lorem message"
    },
    "email": {
      "subject": "Loresm subject",
      "message": "Lorem message"
    }
  }
}
````

Reply:
Done in last commit.